presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -117,6 +120,36 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
 
         TupleDomain<ColumnHandle> oldDomain = handle.getConstraint();
         TupleDomain<ColumnHandle> newDomain = oldDomain.intersect(constraint.getSummary());
```

Comment:
do not initialize this variable here, it is very likely to be overwritten. Let's define it in the same block where `remainingFilter` is defined.

Reply:
i use the variable to select whcih code block i do

Reply:
Right, I overlooked that. Thanks